roles/openshift_logging_eventrouter/defaults/main.yaml

```diff
@@ -2,7 +2,7 @@
 openshift_logging_eventrouter_replicas: 1
 openshift_logging_eventrouter_sink: stdout
 openshift_logging_eventrouter_nodeselector: ""
-openshift_logging_eventrouter_cpu_limit: null
+openshift_logging_eventrouter_cpu_limit: 100m
```

Comment:
This has some logic for 'if defined' in the template.  We should remove this variable from defaults so that logic works correctly.

Reply:
Though, that might break some set_fact logic that is being performed elsewhere...

I think it's fine if we're always going to define it then.  Let's remove the logic from the template.

Reply:
Right, done, this never seem to have worked correctly. Didn't touch other components in this PR though

Reply:
Like other logging components, we dont desire to limit or cap CPU.  This should be reverted and CPU limit only applied if explicitly defined in inventory

Reply:
Wouldn't the template set it to 100m if its undefined? Sounds like the template needs to be reworked then

Reply:
Nothing wrong with the template as i see it https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging_eventrouter/templates/eventrouter-template.j2#L68  we do not want to set it if it is undefined; that's the point.  We desire for it to receive as much CPU is allowed by the kublet by default

Reply:
I think the template if condition needs to be expanded, if you look at the template for our other components we are also checking if its set to `None` and if its empty